src/common/transformations/include/transformations/op_conversions/convert_shapeof3.hpp

```diff
@@ -22,3 +23,9 @@ class ov::pass::ConvertShapeOf3 : public ov::pass::MatcherPass {
     OPENVINO_RTTI("ConvertShapeOf3", "0");
     ConvertShapeOf3();
 };
+
```

Comment:
I think we can add this as gpu specific transform instead of common transform?

Reply:
I moved it from common to GPU plugin as you reviewed. Thank you.